go/host/host.go

```diff
@@ -63,6 +64,7 @@ type host struct {
 	stopControl *stopcontrol.StopControl
 
 	l1BlockProvider hostcommon.ReconnectingBlockProvider
+	l1UpToDate      atomic.Bool                     // Whether the last submitted L1 block was the head L1 block
```

Comment:
`l1UpToDate` -> `l1HeadProcessError` ? Seems more descriptive.

Reply:
It's not about the error, it's about whether our enclave has received the current L1 head.